packages/pilot/src/components/MetricList/style.css

```
@@ -0,0 +1,18 @@
+@import "former-kit-skin-pagarme/dist/styles/spacing.css";
+
+.item {
+  display: flex;
+
+  & > .icon {
+    margin-right: var(--spacing-small);
```

no zeplin a margin entre o icone e o texto está de 8px e aqui está 16px
```suggestion
    margin-right: var(--spacing-tiny);
```

---

@rsmelo boa! Aproveitei também para corrigir o alinhamento vertical entre o ícone e o texto :+1: